straxen/plugins/peaklet_processing.py

```diff
@@ -543,13 +543,21 @@ def compute(self, records_he, start, end):
 
 @export
 @strax.takes_config(
-    strax.Option('s1_max_rise_time', default=110, infer_type=False,
-                 help="Maximum S1 rise time for < 100 PE [ns]"),
-    strax.Option('s1_max_rise_time_post100', default=200, infer_type=False,
+    strax.Option('s1_risetime_area_parameters', default=(50, 80, 12), type=(list, tuple),
```

Comment:
Looking at ` upper_rise_time_area_boundary(area, norm, tau, const)`, `const` should 80 and `tau` should be 12, no?